esphome/components/dac7678/__init__.py

```diff
@@ -0,0 +1,31 @@
+import esphome.codegen as cg
+import esphome.config_validation as cv
+from esphome.components import i2c
+from esphome.const import CONF_ID
+
+CODEOWNERS = ["@NickB1"]
```

Comment:
This will fix compile issues if someone adds the main hub, but no actual outputs
```suggestion
AUTO_LOAD = ["output"]
CODEOWNERS = ["@NickB1"]
```